src/main/java/seedu/address/model/contact/Telegram.java

```diff
@@ -0,0 +1,55 @@
+package seedu.address.model.contact;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+
+/**
+ * Represents a Person's phone number in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidId(String)}
+ */
+public class Telegram {
+
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Message should contain alphanumeric characters and underscore, and it should not be empty";
+
+    public static final String VALIDATION_REGEX = "@" + "[a-zA-Z0-9_]";
```

Comment:
The expression `"[a-zA-Z0-9_]"` only matches exactly one of the characters, so the current regex matches `"@a"` and `"@1"` but not `"@a1"`. A `+` should be added after the square bracket to make the regex match more than 1 of the characters inside.
```suggestion
    public static final String VALIDATION_REGEX = "@" + "[a-zA-Z0-9_]+";
```